packages/@contentful--app-scripts/lib/upload/index.js

```diff
@@ -0,0 +1,15 @@
+const { getUploadSettingsArgs } = require('./get-upload-settings-args');
+const { createAppBundle } = require('./create-app-bundle');
+
+const { buildAppUploadSettings } = require('./build-upload-settings');
+
+module.exports = {
+  async interactive() {
+    const settings = await buildAppUploadSettings();
+    await createAppBundle(settings);
+  },
+  async nonInteractive(program, options) {
```

Comment:
Why do we need `program` here?

Reply:
I used it in a previous implementation, forgot to remove it